runtime/doc/faq.txt

```diff
@@ -0,0 +1,702 @@
+- Disable 'guicursor' in your Nvim config: >vim
+
+    :set guicursor=
+    " Workaround some broken plugins which set guicursor indiscriminately.
+    :autocmd OptionSet guicursor noautocmd set guicursor=
+<
+See also |$TERM| for recommended values of `$TERM`.
+
+
+HOW TO CHANGE CURSOR SHAPE IN THE TERMINAL? ~
+
+- For Nvim 0.1.7 or older: see the note about `NVIM_TUI_ENABLE_CURSOR_SHAPE` in `man nvim`.
+- For Nvim 0.2 or newer: cursor styling is controlled by the 'guicursor' option.
+    - To _disable_ cursor-styling, set 'guicursor' to empty: >vim
+
+            :set guicursor=
+            " Workaround some broken plugins which set guicursor indiscriminately.
+            :autocmd OptionSet guicursor noautocmd set guicursor=
+<
+    - If you want a non-blinking cursor, use `blinkon0`. See 'guicursor'.
+    - 'guicursor' is enabled by default, unless Nvim thinks your terminal doesn't
+        support it. If you're sure that your terminal supports cursor-shaping, set
+        'guicursor' in your |init.vim|, as described in 'guicursor'.
+- The Vim terminal options `t_SI` and `t_EI` are ignored, like all other `t_XX` options.
+- Old versions of libvte (gnome-terminal, roxterm, terminator, ...) do not
+  support cursor style control codes. #2537
+  https://github.com/neovim/neovim/issues/2537
+
+
+HOW TO CHANGE CURSOR COLOR IN THE TERMINAL? ~
+
+Cursor styling (shape, color, behavior) is controlled by 'guicursor', even in
+the terminal. Cursor color (as opposed to shape) only works if
+'termguicolors' is set.
+
+'guicursor' gives an example, but here's a more complicated example
+which sets different colors in insert-mode and normal-mode:
+>vim
+    :set termguicolors
+    :hi Cursor guifg=green guibg=green
+    :hi Cursor2 guifg=red guibg=red
+    :set guicursor=n-v-c:block-Cursor/lCursor,i-ci-ve:ver25-Cursor2/lCursor2,r-cr:hor20,o:hor50
+<
+
+CURSOR STYLE ISN'T RESTORED AFTER EXITING OR SUSPENDING AND RESUMING NVIM ~
+
+Terminals do not provide a way to query the cursor style. Use autocommands to
+manage the cursor style:
+>vim
+    au VimEnter,VimResume * set guicursor=n-v-c:block,i-ci-ve:ver25,r-cr:hor20,o:hor50
+      \,a:blinkwait700-blinkoff400-blinkon250-Cursor/lCursor
+      \,sm:block-blinkwait175-blinkoff150-blinkon175
+
+    au VimLeave,VimSuspend * set guicursor=a:block-blinkon0
+<
+
+CURSOR SHAPE DOESN'T CHANGE IN TMUX ~
+
+tmux decides that, not Nvim. See |tui-cursor-shape| for a fix.
+
+See #3165 https://github.com/neovim/neovim/pull/3165 for discussion.
+
+
+CURSOR FLICKER IN TMUX? ~
+
+If cursor `_` appears and disappears very quickly when opening nvim without a
+document under tmux, and you set |ctermbg| in `EndOfBuffer` and `Normal`, try
+setting these to `NONE`:
+>vim
+    hi EndOfBuffer ctermbg=NONE ctermfg=200 cterm=NONE
+    hi Normal ctermbg=NONE ctermfg=200 cterm=NONE
+<
+
+HOW TO USE THE WINDOWS CLIPBOARD FROM WSL? ~
+
+Try the suggestion given by |clipboard-wsl|:
+>vim
+    let g:clipboard = {
```

Comment:
assuming we still want this faq item, we can just link to `|clipboard-wsl|` and don't need to repeat the code sample here.

Reply:
If in doubt, remove. I've done so for obviously outdated content but erred on the side of caution.